metrics.nim

```diff
@@ -979,43 +979,49 @@ when defined(metrics):
 
 when defined(metrics):
   {.pop.} # raises - no matter what, can't annotate async methods
-  import asynchttpserver, asyncdispatch
+  import chronos, chronos/apps
```

Comment:
assuming more apps appear, seems better to import just the server?

Reply:
in particular, will this pull in bearssl?

Reply:
Of course.